docs/2/general/faq/security-faq.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: docs
+title: {Microshar Data Security and Privacy FAQs}
+description: {Frequently asked questions regarding Microshare data security and privacy}
+toc: true
+---
+
+{% include image.html url="/assets/img/thumbnail-8.jpg" height="900" width="900" description="thumbnail 2" %}
+
+
+<br>
+
+---------------------------------------
+
+<!-- ********************
+**    New Question     **
+********************* -->
+
+{% capture title %}
+Where can I learn more about the data security practices of Microshare?
+{% endcapture %}
+
+{% capture content %}
+For information the the data security practices of Microshare, please visit our Data Security and Privacy Trust Center at https://www.microshare.io/microshares-trust-center-data-security/
+{% endcapture %}
+
+{% include accordion.html title=title content=content key="security" %}
+
+
+<!-- ********************
+**    New Question     **
+********************* -->
+
+{% capture title %}
+Is the Microshare application compliant with relevant data protection and privacy regulations, such as GDPR or SOC 2?
+{% endcapture %}
+
+{% capture content %}
+Microshare is SOC 2, IASME Cyber Essentials, and IASME Cyber Assurance (GDPR) complaint. We undergo annual third-party assessments as part of the recertification process to ensure we meet all data protection and privacy requirements. For more information, please visit the Microshare Data Security and Privacy Trust Center at https://www.microshare.io/microshares-trust-center-data-security/
+{% endcapture %}
+
+{% include accordion.html title=title content=content key="security" %}
```

Comment:
the "key" elements should all be different.

Reply:
Addressed.